charts/tidb-cluster/values.yaml

```diff
@@ -173,10 +173,6 @@ pd:
     # # when the kubelet is configured to allow unsafe sysctls
     # - name: net.core.somaxconn
     #   value: "32768"
-    # - name: net.ipv4.tcp_syncookies
-    #   value: "0"
```

Comment:
Remove them because they cannot be set in containers

Reply:
This should still work by default if the kubelet has the unsafe sysctl enabled. When not enabled, users need to add an annotation to set this. So I don't think we should remove it.

Reply:
No, they are not for containers.